the-blue-alliance-ios/View Controllers/Settings/SettingsTableViewController.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  SettingsTableViewController.swift
+//  The Blue Alliance
+//
+//  Created by Anas Merbouh on 18-05-20.
+//  Copyright © 2018 The Blue Alliance. All rights reserved.
+//
+
+import UIKit
+
+class SettingsTableViewController: UITableViewController {
+    
+    // MARK: - View's lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "BaseCell")
+        tableView.register(UINib(nibName: "SwitchTableViewCell", bundle: nil), forCellReuseIdentifier: "SwitchCell")
+    }
+    
+    // MARK: - Interface Methods
+    
+    @IBAction func doneTapped(_ sender: UIBarButtonItem) {
+        self.dismiss(animated: true, completion: nil)
+    }
+    
+    // MARK: - Table view configuration
+    
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 4
+    }
+    
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section {
+        case 0:
+            return "Account"
+            
+        case 1:
+            return "Notifications"
+            
+        case 2:
+            return "Info"
+            
+        case 3:
+            return "Extra"
+            
+        default:
+            fatalError("This section does not exist")
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        switch section {
+        case 0:
+            return 1
+            
+        case 1:
+            return 1
+            
+        case 2:
+            return 4
+            
+        case 3:
+            return 2
+            
+        default:
+            fatalError("This section does not exist")
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "BaseCell", for: indexPath)
```

Comment:
The cell switching logic here is pretty weird - can we take the pattern from the Team Info and apply the same sort of thing here? https://github.com/the-blue-alliance/the-blue-alliance-ios/blob/swift/the-blue-alliance-ios/View%20Controllers/Teams/TeamInfoTableViewController.swift#L86